### PR TITLE
Move GRIP to wpilib-tools

### LINKED ISSUE #505: Move GRIP docs to Software Tools section

### PR DESCRIPTION
**Pull request by ItayZiv**
fixes #505.
WIP because I also want to fix a few stuff in the docs, like references to Eclipse and some of the docs seem to have been written before GRIP was released and mention that.

### CODE REVIEW

**Comment by ItayZiv** [comment]
I'll remove it for now, who originally wrote the documentation, since adding stuff like references to the NT publishing, and I have a few other questions.
Also, what is the current "Status of GRIP"?

**Comment by Daltz333** [comment]
GRIP is maintained by the WPI (not WPILib) team. Consider it supported for right now.

File: source/docs/software/vision-processing/index.rst [hunk 1]
```diff
@@ -5,5 +5,4 @@ Vision Processing
    :maxdepth: 2
 
    introduction/index
```
Comment (AustinShalit) [inline source/docs/software/vision-processing/index.rst:7]: I think we need something in the vision section that mentions GRIP. Can we add a ¶ to the intro?

**Comment by AustinShalit** [comment]
Hi @ItayZiv.  @sciencewhiz suggested yesterday in the issue that it might actually be better to leave GRIP in vision processing and remove the tools section: https://github.com/wpilibsuite/frc-docs/issues/505#issuecomment-609141406. Do you want to close this PR and open a new one that does that instead?

**Comment by ItayZiv** [comment]
Made a new PR, https://github.com/wpilibsuite/frc-docs/pull/604.
Closing this one.

**Pull request closed without merging**